### ISSUE TITLE
Configuration file "file_path" not found

### ISSUE THREAD

**Issue by habin201291** [warp-tech/warpgate#955]
Hi,

I created a TOML file named Settings in the `/warpgate-protocol-ssh/src/server` directory and used [config-rs](https://github.com/mehcode/config-rs) to get the values from that file.

I ran it with the debug build and it worked fine, but when I built the release build and ran it on Production, I got the following error:
`called Result::unwrap() on an Err value: configuration file "warpgate-protocol-ssh/src/server/Settings" not found`.

Hope this help!

**Comment by Eugeny** [comment]
Can you explain what you're trying to do?

**Comment by habin201291** [comment]
I am building from source step by step following the Contributing/building from source guide and also adding some of my own features. I have created an additional file named Settings.toml in the warpgate-protocol-ssh/src/server directory. I used config-rs to get the values in the Settings.toml file from the warpgate-protocol-ssh/src/server/session.rs file.

My code is as follows:
`let settings = Config::builder()`
`.add_source(config::File::with_name("warpgate-protocol-ssh/src/server/Settings"))`
`.add_source(config::Environment::with_prefix("APP"))`
`.build()`
`.unwrap();`

I am not sure if I need to deploy the source code to the Production server.
